VTEX - Marketplace APIs - Suggestions.json

```diff
@@ -1635,6 +1714,70 @@
                     }
                 }
             },
+            "responseGetAccountsMatcherSettings": {
+                "title": "GetAccountsMatcherSettings",
+                "description": "Response with information on current matcher scoring configuration, rules, specifications mapping and matchFlux.",
+                "type": "object",
+                "properties": {
+                    "Score": {
+                        "description": "Matcher rates received SKUs by comparing the data sent by sellers to existing fields in the marketplace. The calculation of these scores determines whether the product has been: `Approved` or `Denied`.",
+                        "type": "object",
+                        "properties": {
+                            "Approve": {
+                                "type": "integer",
+                                "description": "Minimum approval score set by the marketplace."
+                            },
+                            "Reject": {
+                                "type": "integer",
+                                "description": "Bounce score set by marketplace."
+                            }
+                        }
+                    },
+                    "Matchers": {
+                        "description": "Array of objects that presents a list of matchers configured in the marketplace. If the marketplace has more than one matcher configured in the account, an array will be displayed for each matcher.",
```

Comment:
Adicionaria que "by default, the VTEX Matcher is set up automatically in VTEX accounts and has the XYZ matcherId". Aí checa com os devs qual o matcherId do Matcher da VTEX pra já indicar